### DISCUSSION TITLE
New-Page as name invalid?

### DISCUSSION THREAD

**Discussion by cinatic** [requarks/wiki#3988] [Help / Questions]
If i create a page and do not alter the preset page name then i will get an "page does not exist" browsing it

is this expected behaviour?

**Comment by skledarg** [comment]
I can't reproduce this on Wikijs 2.5.201, could you add more details? What version are you using and what **Title** and **path** does the newly created page have when you create it?

**Reply by cinatic** [reply]
strange, after i recreated the tree, it works, so it was only a coincidence that some pages has been created and some not
